wandb/sdk/wandb_login.py

```diff
@@ -274,6 +265,30 @@ def prompt_api_key(self) -> None:
         self._key = key
 
 
+def _verify_login() -> bool:
+    from . import wandb_setup
```

Comment:
This looks overly cautious to me. Could this import go in the import block at the top?

I think since it's an import of a module and not a `from <module> import <symbol>` import, there can't be a cycle.